client/my-sites/plugins/plugin-remove-button/index.jsx

```diff
@@ -19,32 +19,30 @@ import PluginAction from 'calypso/my-sites/plugins/plugin-action/plugin-action';
 import { removePlugin } from 'calypso/state/plugins/installed/actions';
 import { isPluginActionInProgress } from 'calypso/state/plugins/installed/selectors';
 import { removePluginStatuses } from 'calypso/state/plugins/installed/status/actions';
+import { getPluginActionDailogMessage } from '../utils';
 
 import './style.scss';
 
 class PluginRemoveButton extends Component {
 	static displayName = 'PluginRemoveButton';
 
 	removeAction = () => {
+		const { translate, plugin, site } = this.props;
+		const dialogOptions = {
+			additionalClassNames: 'plugins__confirmation-modal',
+			isScary: true,
+		};
+		const heading = translate( 'Remove %(pluginName)s', {
+			args: {
+				pluginName: plugin.name,
+			},
+		} );
 		accept(
-			this.props.translate(
-				'Are you sure you want to remove {{strong}}%(pluginName)s{{/strong}} from' +
-					' %(siteName)s? {{br /}} {{em}}This will deactivate the plugin and delete all' +
-					' associated files and data.{{/em}}',
```

Comment:
I can see that we removed the "This will deactivate the plugin and delete all associated files and data." part, but IMO it's an important one, and we should keep it. What do you think?

Reply:
I realized this part is displayed, but here's how I missed it: https://github.com/Automattic/wp-calypso/pull/67799#pullrequestreview-1107285580

Reply:
I chatted with @grbicsanja about the same and replaced it based on the text she provided.

Reply:
But, I think it makes sense to keep this. What do you say, @grbicsanja?